demo/src/screens/componentScreens/ChipsInputScreen.tsx

```diff
@@ -1,5 +1,5 @@
 import React, {Component} from 'react';
-import {View, Text, Card, TextField, Button, Colors, ChipsInput} from 'react-native-ui-lib'; //eslint-disable-line
+import {View, Text, Colors, ChipsInput} from 'react-native-ui-lib'; //eslint-disable-line
```

Comment:
Same here, you can remove the `//eslint-disable-line`

Reply:
Fixed